### ISSUE TITLE
Possible MATLAB 2019a incompatibility

### ISSUE THREAD

**Issue by mattzett** [gemini3d/mat_gemini#9]
```
>> plotall(direc,{'png','eps'},[],[],0)
sim grid dimensions: 744  756  210
Full or milestone input detected.
../simulations/mooreOK3D_hemis_medres_corrected/20130520_71100.000001.h5 => plot3D_curv_frames_long
Error using warning
The message must be specified as either a character vector, string scalar, or a message structure.

Error in plotframe (line 127)
    warning(excp)

Error in plotall (line 63)
      plotframe(direc, params.times(i), saveplot_fmt, plotfun, xg, h)

**Comment by mattzett** [comment]
This seems to not be an issue for R2020a...

**Comment by scivision** [comment]
fixed by c53525f
